lichess/api.py

```diff
@@ -321,6 +321,11 @@ def tv_channels(**kwargs):
     """Wrapper for the `GET /tv/channels <https://github.com/ornicar/lila#get-tvchannels-fetch-current-tournaments>`_ endpoint."""
     return _api_get('/tv/channels', kwargs)
 
+def cloud_evaluation(**kwargs):
+    """Wrapper for the `GET api/cloud-eval` endpoint.
+    Entry data: fen, multiPv = 1 (numver of variations), variant = "standard". """
```

Comment:
Use a link to the api (as in other docs) rather than explicitly documenting the parameters. The idea being that it may easily become out of date.